src/gemm_v2.cpp

```diff
@@ -36,10 +36,14 @@
 #endif
 
 #if MIOPEN_USE_ROCBLAS
+#if defined(_WIN32) && (HIP_PACKAGE_VERSION_FLAT < 5007000000ULL)
```

Comment:
```suggestion
#if defined(_WIN32) || (HIP_PACKAGE_VERSION_FLAT < 5007000000ULL)
```

Reply:
@apwojcik  Would you agree? Perhaps I am missing something.

Reply:
There was no check before and on Linux Beta Feature API was always on. I made it optional on Windows and available only for HIP SDK 5.7, but not for HIP SDK 5.5 (which is officialy released on Windows and there is no newer available).

Reply:
I can change as you suggesting so all using older HIP SDK will compile on Linux, too.